src/js/actions/layers.js

```diff
@@ -1553,7 +1553,7 @@ define(function (require, exports) {
             return layer.isBackground;
         });
         
-        options = _.merge({ coalesce: false }, options);
+        options = _.merge({}, options);
```

Comment:
Sorry, I should've mentioned this to avoid confusion. these default options were added by me while I was working on another issue, and apparently we don't need them because we use `!!options. coalesce` to deal with `undefined` case below.